recipes/hepdata-lib/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "hepdata-lib" %}
+{% set version = "0.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hepdata_lib-{{ version }}.tar.gz
+  sha256: 88325da7e9ee9408eafebb518117ff3c39f438048766f1a19f59ef22f6c6d1d4
+
+build:
+  # Will fail on Windows as imagemagick does not support Windows builds, but
+  # failing on a noarch is preferred by the conda-forge maintainers
+  noarch: python
```

Comment:
The Windows build will fail, but that is expected, and so leaving this as is given https://github.com/conda-forge/staged-recipes/pull/25702#discussion_r1521569904.